packages/make-styles-webpack-loader/src/webpackLoader.test.ts

```diff
@@ -161,6 +162,28 @@ function testFixture(fixtureName: string, options: CompileOptions = {}) {
   });
 }
 
+describe('shouldTransformSourceCode', () => {
```

Comment:
nit: you never test the `false` codepath

Reply:
Added assertions 👍